src/IconInner.js

```diff
@@ -1,16 +1,29 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import DOMPurify from 'dompurify';
+import { JSDOM } from 'jsdom';
 // for now this icons json is generated via the build script from latest feather
 // TODO: automatically generate this JSON via this repo's build script
 import icons from './icons.json';
 
 class IconInner extends PureComponent {
   createMarkup(markup) {
+
     // sanitize markup first:
-    const sanitizedMarkup = DOMPurify.sanitize(markup);
+    let sanitizedMarkup = DOMPurify.sanitize(markup);
```

Comment:
instead of using `let` let's just create a separate function `sanitizeMarkup(markup)` here, and conditionally return based on whether `window` exists like below. Does that make sense?

Reply:
That totally makes sense. I just commited a change. It should be fine now